File: QuanLyKhachSanApp/QuanLyKhachSanApp/Models/PhieuNhapKho.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace QuanLyKhachSanApp.Models
{
    [Table("PhieuNhapKho")]
    public class PhieuNhapKho
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Required]
        public int PhieuNhapID { get; set; }

        [Column(TypeName = "Date")]
        public DateTime? NgayNhap { get; set; }

        public double? TongTien { get; set; }

        public int? NhanVienID { get; set; }

        [ForeignKey("NhanVienID")]
        public virtual NhanVien NhanVien { get; set; }

        [InverseProperty("PhieuNhapKho")]
        public virtual ICollection<ChiTietPhieuNhap> ChiTietPhieuNhap { get; set; }

    }
}

File: QuanLyKhachSanCli/src/models/PhieuNhapKho.ts
import { ChiTietPhieuNhap } from "@/models/ChiTietPhieuNhap";
import { NhanVien } from "@/models/NhanVien";

export interface PhieuNhapKho {
    PhieuNhapID: number;
    NgayNhap: Date;
    TongTien: number;
    NhanVienID: number;
    NhanVien?: NhanVien;
    ChiTietPhieuNhap?: ChiTietPhieuNhap[];
}
